packages/react/src/accordion/Accordion.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, forwardRef, type Ref, Children, useId } from 'react';
+import { cx } from 'cva';
+import { ChevronDown } from '@obosbbl/grunnmuren-icons-react';
+import { Heading } from '../content';
+
+type AccordionProps = {
+  children: React.JSX.Element[] | React.JSX.Element;
+
+  /** Additional CSS className for the element. */
+  className?: string;
+
+  /** Additional style properties for the element. */
+  style?: React.CSSProperties;
+};
+type AccordionItemProps = {
+  children: React.JSX.Element[];
+
+  /** Additional CSS className for the element. */
+  className?: string;
+
+  /** Additional style properties for the element. */
+  style?: React.CSSProperties;
+};
+
+function Accordion(props: AccordionProps, ref: Ref<HTMLDivElement>) {
+  return (
+    <div
+      className={cx('flex flex-col rounded-lg bg-white p-2', props.className)}
+      ref={ref}
+    >
+      {props.children}
+    </div>
+  );
+}
+
+function AccordionItem(props: AccordionItemProps, ref: Ref<HTMLDivElement>) {
+  const [open, setOpen] = useState(false);
+  const contentId = useId();
+
+  return (
+    <div
+      className={cx(
+        'group relative px-2',
+        /** Pseudoelement for the gray bottom border */
+        'after:absolute after:left-[9px] after:right-[9px] after:h-px after:bg-gray-light after:last:h-0',
+      )}
+      data-open={open || undefined}
+      ref={ref}
+    >
+      {Children.map(props.children, (child) => {
+        /** Since we are using <Heading> and <Content> from Grunnmuren to
+         * render the accordion item, we can use the level-prop to identify the heading component */
+        if (child.props.level) {
+          return (
+            <Heading level={child.props.level}>
+              <button
+                onClick={() => setOpen(!open)}
+                className={cx(
+                  'flex w-full flex-1 items-center justify-between rounded-md py-[18px] text-left font-semibold leading-7',
+                )}
+                aria-expanded={open}
+                aria-controls={contentId}
+              >
+                {child.props.children}
+                <ChevronDown
+                  className={cx(
+                    'text-base transition-transform duration-150 group-data-[open]:rotate-180 motion-reduce:transition-none',
+                  )}
+                />
+              </button>
+            </Heading>
+          );
+        } else {
+          return (
+            <div
+              aria-labelledby={contentId}
```

Comment:
Denne må endres til:
```suggestion
              id={contentId}
```